site/documentation/content/admin-guide/mongodb-device-registry-config.md

```diff
@@ -16,6 +16,12 @@ The registry is implemented as a Quarkus application, using a MongoDB database a
 either directly from the command line or by means of starting the corresponding
 [Docker image](https://hub.docker.com/r/eclipse/hono-service-device-registry-mongodb/) created from it.
 
+The registry is compatible and known to work with following MongoDB versions:
+
+* [MongoDB 4.4](https://www.mongodb.com/docs/v4.4/release-notes/4.4/)
+* [MongoDB 5.0](https://www.mongodb.com/docs/v5.0/release-notes/5.0/)
+* [MongoDB 6.0](https://www.mongodb.com/docs/v6.0/release-notes/6.0/)
```

Comment:
while I believe that your recent (manual) tests were successful in this regard, I wonder how we want to make sure that this is (still) true when we make changes to the registry ...

Reply:
It comes to my mind two options:
- have written rule somewhere if we modify the registry call integration tests against all supported mongodb versions
- create maven pom profiles that set different mongo versions, eventually automate calling all profiles during integration tests

Reply:
FMPOV the latter sounds much more promising :-)

Reply:
Maybe you can extend the integration tests in another PR ...